docs/README.md

```diff
@@ -21,13 +21,13 @@ These accounts serve as an interface for code running somewhere on your server.
 #### Installation
 
 ```bash
-$ npm install telegraf --save
+npm install telegraf --save
```

Comment:
https://wiki.archlinux.org/index.php/Help:Reading#Regular_user_or_root

Reply:
The `$` is added back with CSS:

https://github.com/telegraf/telegraf/blob/bcbe8559c16267166073838c1ed560a010c98a34/docs/theme/assets/css/custom.css#L2-L8

@YouTwitFace did it to make the `$` unselectable and thus uncopyable.